### PR TITLE
ensure times.txt goes to outputDir

### PR DESCRIPTION
**Pull request by joekiller**
Makes sure that times are recorded where output is expected.

### CODE REVIEW

File: scripts/8/run.sh [hunk 1]
```diff
@@ -113,7 +113,7 @@ do
         adb -s $device shell am instrument -w -e class $line $package.test/android.support.test.runner.AndroidJUnitRunner > $runningTest
         endTime=$(node -e 'console.log(Date.now())')
         echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis."
-        echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis." >> "artifacts/times.txt"
+        echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis." >> "$outputDir/times.txt
```
Comment (sromku) [inline scripts/8/run.sh]: Cool. Just missing - " in the end.

File: scripts/9/run.sh [hunk 2]
```diff
@@ -113,7 +113,7 @@ do
         adb -s $device shell am instrument -w -e class $line $package.test/android.support.test.runner.AndroidJUnitRunner > $runningTest
         endTime=$(node -e 'console.log(Date.now())')
         echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis."
-        echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis." >> "artifacts/times.txt"
+        echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis." >> "$outputDir/times.txt
```
Comment (sromku) [inline scripts/9/run.sh]: missing - " in the end

File: scripts/run.sh [hunk 3]
```diff
@@ -113,7 +113,7 @@ do
         adb -s $device shell am instrument -w -e class $line $package.test/android.support.test.runner.AndroidJUnitRunner > $runningTest
         endTime=$(node -e 'console.log(Date.now())')
         echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis."
-        echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis." >> "artifacts/times.txt"
+        echo "test ($line) device ($device) , duration: $((endTime-startTime)) millis." >> "$outputDir/times.txt
```
Comment (sromku) [inline scripts/run.sh]: missing - " in the end

File: scripts/8/run.sh (hunk 1 above)
Comment (joekiller) [reply scripts/8/run.sh]: Sorry fixed.

**Comment by joekiller** [comment]
Fixed those missing quotes.

**Comment by sromku** [comment]
Great! Thanks for the fix.

**Pull request merged**